gen/builders/builder_tipset.go

```diff
@@ -218,7 +224,7 @@ func (b *TipsetVectorBuilder) CommitApplies() {
 		//  we don't get the intermediate roots through null rounds, so this
 		//  is not straightforward.
 		//  https://github.com/filecoin-project/test-vectors/issues/91
-		b.Rewards.RecordAt(ts.Epoch)
+		b.Rewards.RecordAt(ts.EpochOffset)
```

Comment:
I am a bit confused... why is this `ts.EpochOffset` and not `execEpoch`?

Reply:
Because exec epoch will be different at every variant that the builder is being run for. This could do with some refinement, though.

We could either:
1. record the real execution epoch (`v.ProtocolVersion.Epoch + InitialEpoch + EpochOffset`), but then `b.Rewards.ForEpoch()` would need to perform the same calculation. At the end of the day, it requires more work and they cancel out, so not worth IMO.
2. keep recording at the epoch offset, but rename `b.Rewards.ForEpoch` to `b.ForEpochOffset`, which makes things clearer.

I'm gonna go with 2.